internal/cmd/config.go

```diff
@@ -118,6 +126,12 @@ func LoadConfig(env Env) (config *Config, err error) {
 			return
 		}
 
+		if env[DIRENV_IGNORE_KEYS] != "" {
+			config.EnvIgnoreKeys = strings.Split(env[DIRENV_IGNORE_KEYS], ",")
+		} else {
```

Comment:
The config option is enough. Historically direnv was configured with env vars but that's more of an artefact now.